4337/contracts/test/TestUniqueSigner.sol

```diff
@@ -3,6 +3,16 @@
 pragma solidity >=0.8.0;
 
 import {ISignatureValidator} from "@safe-global/safe-contracts/contracts/interfaces/ISignatureValidator.sol";
+import {IUniqueSignerFactory} from "./SafeSignerLaunchpad.sol";
+
+function checkSignature(bytes memory data, uint256 signature, uint256 key) pure returns (bytes4 magicValue) {
+    uint256 message = uint256(keccak256(data));
+
+    // A very silly signing scheme where the `message = signature ^ key`
```

Comment:
almost a one-time pad: https://en.wikipedia.org/wiki/One-time_pad 😆

Reply:
Except the key is public :clown_face:. :1st_place_medal: cryptography here we go!